### ISSUE TITLE
Rotate by 90 degrees and scroll view cannot scroll to full width of image

### ISSUE THREAD

**Issue by shimeny** [kishikawakatsumi/PEPhotoCropEditor#75]
When i try to rotate an image to 90 degrees, the image bounds are larger than the scrollview bounds and it seems that you cannot include the edges of the image in the crop

**Comment by hermeneutic** [comment]
Hello there,

did you already solve your problem?

**Comment by shimeny** [comment]
Not exactly, i found that the logic is floored and i came across many other scrollview examples that had the same problem. That is until i found Photo Tweaks a newly added repository, i haven't had a chance yet to review the code in great detail but it does achieve what i need

**Comment by hermeneutic** [comment]
thank for the info. yes, that is exactly what i need too.. except it looks like don't have keepCropAspectRatio.. :)

**Comment by shimeny** [comment]
Just out of interest what are you trying to achieve? I am developing a photo editor that replicates the ios 8 editor

**Comment by hermeneutic** [comment]
i only need crop controller like in iOS 8.. which detect image bounds.. :)
if you can help me, please contact me by email.

thanks.
